tensorflow/lite/g3doc/examples/audio_classification/overview.md

```diff
@@ -26,10 +26,10 @@ If you are new to TensorFlow Lite and are working with Android, we recommend
 exploring the following example applications that can help you get started.
 
 You can leverage the out-of-box API from
-[TensorFlow Lite Task Library](../../inference_with_metadata/task_library/audio_classifier)
+[TensorFlow Lite Task Library](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/inference_with_metadata/task_library/audio_classifier.md)
```

Comment:
Can you keep the relative path here? Appending .md is sufficient. Thanks!

Reply:
@sirakiin 
As you suggested I added the relative path.

Thanks!